packages/fs-routes/test/test.ts

```diff
@@ -5,22 +5,42 @@ const testDir = path.resolve(__dirname, '..', 'test-dir');
 
 function assertRoutes(routes) {
   const output = [
+    {
+      path: testDir + '/home.cjs',
+      route: '/home',
```

Comment:
hmm, if there are 2 files next to each other with the same name but different extensions, does node use cjs over js?  if so, i would expect to not see the js routes here.

Reply:
Good feedback.
We can define what the expected result will be.
I should expect in a project you use 1 of both extensions, but we can filter out double matches.
I think if there are files with the same name, .js should be there default.

But if you have a better solution that we can use ES module routes, that will be also a lot better.
The require() in openapi-framework that dynamic load the files should use import().
It is for me to much refactoring or to less code knowledge, but if you can use import, that will be a better fix.

Reply:
what does node do?  i don't feel you answered my previous question.

Reply:
.js is the default of you don't add an extension.
But I'm this package all files are found with glob, and than all files are used.